index.html

```diff
@@ -5648,32 +5661,33 @@ <h3>Language and Base Direction</h3>
         </p>
 
         <p>
-Implementers utilizing JSON-LD are strongly urged to
-<a href="#extensibility">extend</a> the JSON-LD Context defining the
-internationalized <a>property</a> and use the Scoped Context feature of JSON-LD
-to alias the <code>@value</code>, <code>@language</code>, and
-<code>@direction</code> keywords to <code>value</code>, <code>lang</code>,
-and <code>dir</code>, respectively. An example of a JSON-LD Context snippet
-doing this is shown below.
+Multiple language value objects MAY be provided as an array value for the
+property:
         </p>
 
-        <pre class="example nohighlight" title="Specifying scoped aliasing for language information">
-"title": {
-  <span class="highlight">"@context": {"value": "@value", "lang": "@language", "dir": "@direction"}</span>,
-  "@id": "https://www.w3.org/2018/credentials/examples#title"
-}
+        <pre class="example nohighlight" title="Multiple language texts provided for title">
+"title": [
+  {
+    "@value": "<span class="highlight">HTML and CSS: Designing and Creating Websites</span>",
+    "@language": "<code>en</code>"
+  },
+  {
+    "@value": "<span class="highlight">HTML و CSS: تصميم و إنشاء مواقع الويب</span>",
+    "@language": "<code>ar</code>",
+    "@direction": "<code>rtl</code>"
+  }
+]
         </pre>
-
       </section>
 
       <section class="informative">
         <h3>Complex Language Markup</h3>
 
         <p>
-When multiple languages, base directions, and annotations are used in a single
-natural language string, more complex mechanisms are typically required. It is
-possible to use markup languages, such as HTML, to encode text with multiple
-languages and base directions. It is also possible to use the
+When a single natural language string contains multiple languages, base
+directions, and annotations, a more complex mechanism is typically required.
+It is possible to use markup languages, such as HTML, to encode text with
+multiple languages and base directions. It is also possible to use the
```

Comment:
The text here about base direction is wrong. Any paragraph--even one in multiple languages and scripts--has only one base paragraph direction. Inline directional changes that are part of the paragraph do exist and can require care on the part of the author, but this is outside the scope of string metadata. Perhaps:

```suggestion
When a single natural language string contains multiple languages or annotations,
the contents of the string might require additional structure or markup in order to be
presented correctly. It is possible to use markup languages, such as HTML, to label spans of text in different languages or to supply string-internal markup needed for proper display of <a>bidirectional text</a>. It is also possible to use the
```